### PR TITLE
fix column when tabs are used

### LINKED ISSUE #146: Wrong column when using tabs
The range for the errors is set as
```
startPoint = [message.line - 1, message.column - 1]
endPoint = [message.line - 1, message.column]
```

The second element is set as the column of the error but the range thinks it is the character of the line.

When using tabs the column of a line is not the same as the character.

image

The `t` is underlined because it is the 3rd character of the line but the `"` should have been underlined as it is in the 3rd column

### PR DESCRIPTION
**Pull request by UziTech**
closes #146

### CODE REVIEW

**Comment by UziTech** [comment]
I was going to include the `rangeFromLineNumber` in this pull request but that broke some tests. plus since it isn't related to the tab bug it should probably be on its own commit

**Comment by Arcanemagus** [comment]
Hmm, not sure how this got missed before, looking over it now.

File: lib/main.coffee [hunk 1]
```diff
@@ -101,8 +101,24 @@ module.exports =
             return [] unless result.files[filePath]
             messages = result.files[filePath].messages
           return messages.map (message) =>
-            startPoint = [message.line - 1, message.column - 1]
-            endPoint = [message.line - 1, message.column]
+
+            # fix column in line with tabs
+            column = message.column
+            if @tabWidth > 1
```
Comment (UziTech) [inline lib/main.coffee]: if `@tabwidth = 1` than the character column will be same as the screen column so there is no need to calculate it.

The column returned by PHPCS is only incorrect when `@tabwidth > 1`

Comment (Arcanemagus) [inline lib/main.coffee]: Heh, that makes sense as well. I guess there is no point in doing the extra work if it's going to come up with the same result 😉.

**Comment by Arcanemagus** [comment]
Published in v1.4.1. 🎉

**Pull request merged**